hash/ibm5170.xml

```diff
@@ -15481,6 +15481,24 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="tetriscw">
+		<description>Tetris Classic (Windows 3.x release)</description>
+		<year>1992</year>
+		<publisher>Spectrum Holobyte</publisher>
+		<info name="developer" value="Sphere" />
+		<info name="version" value="Version 1.0 10/23/92" />
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="1474560">
+				<rom name="Tetris Classic (windows 3.x release) [Spectrum Holobyte] [1992] [3.5DD] [Disk 2 of 2].img" size="1474560" crc="79804dc4" sha1="2ce4e7641a5d9422d79a85d300a7c0f9140ade88"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="Tetris Classic (windows 3.x release) [Spectrum Holobyte] [1992] [3.5HD] [Disk 1 of 2].img" size="737280" crc="655bda8e" sha1="2a2053439e2bec82cb0be8cd4bae8035e9cf33a7"/>
```

Comment:
Disk 1 and 2 are the wrong way around?

Reply:
Sorry, yes they are, my mistake. Can you switch them? please.

Reply:
Ok it's done. It is something you could have done too by simply doing an additional commit to the PR.